aten/src/ATen/native/Distance.cpp

```diff
@@ -22,6 +23,32 @@ Tensor pdist(const Tensor& self, const double p) {
   return at::_pdist_forward(self.contiguous(), p);
 }
 
+Tensor cdist(const Tensor& x1, const Tensor& x2, const double p) {
+    AT_CHECK(x1.dim() == 2, "cdist only supports 2D tensors, X1 got: ", x1.dim(), "D");
```

Comment:
you still have 4-spacing at the start of the function (you fixed the cases that are indented from here, though).

Reply:
Fixed